### PR TITLE
fix workflows count metric

### PR DESCRIPTION
**Pull request by danielnorberg**

### CODE REVIEW

**Comment by honnix** [comment]
This is nice. ^

Test case please... ;)

**Comment by danielnorberg** [comment]
@honnix added a test =)

File: styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java [hunk 1]
```diff
@@ -377,7 +377,7 @@ void tickTriggerManager() {
 
   private void warmUpCache(WorkflowCache cache, Storage storage) {
     try {
-      storage.workflowsWithNextNaturalTrigger().keySet().forEach(cache::store);
+      storage.workflows().values().forEach(cache::store);
```
Comment (honnix) [inline styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java]: not sure we really want to cache everything including those not initialised.

**Comment by honnix** [comment]
You didn't change that metric (all workflows)?

**Comment by honnix** [comment]
Sorry that's done via the cache. Now I understand why changing that part. 

👍

File: styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java (hunk 1 above)
Comment (honnix) [inline styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java]: disregard my comment.

**Pull request merged**